release/RELEASE.md

```diff
@@ -0,0 +1,80 @@
+# Release process
+
+This document will cover process and tips regarding foreman-docker releases.
+It should serve as a reference to the person doing the release, however the project should
+strive to automate as much as possible of this process.
+
+## Versioning
+
+`foreman-docker` follows [Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html). This means:
+* MAJOR versions: break compatibility with the latest supported Foreman version. For instance, foreman-docker 1.0 breaks Foreman 1.6 compatibility.
+* MINOR versions: enhance foreman-docker with features in a backwards-compatible manner.
+* PATCH versions: contain bugfixes for the latest minor version in a backwards-compatible manner.
+
+`foreman-docker` will always contain a git tag for each specific version, x.y.z.
+Major versions are maintained in `-stable` branches. As an example, when 2.0 is released, it goes in
+the 2.0-stable branch. When 2.1.0 is released, the 2.0-stable branch is updated to include the changes.
+
+## Frequency
+
+Releases are done weekly unless there were not any commits merged in between releases.
+
+## Tips before releasing
+
+* Test against the current Foreman version. For instance, the person releasing 1.0.2, should
+ensure it works well with Foreman 1.7 as that is the version supported by 1.y.z releases.
+* Always test using Foreman and Katello, as Katello will depend on `foreman-docker` for some tasks.
+Use the latest Katello version that is stable with the Foreman version supported.
+* Make sure there are no discrepancies in the dependencies specified in the gemspec versus the
+dependencies of the RPM spec and deb sources.
+* If any dependency in the gemspec was updated, and the package in foreman-packaging does not serve
+it, update the package or consider dropping the dependency.
+* Test tagging, branching, and pushing changes to your local fork first. Only if it looks then you
+should do the same in the project repository.
+
+These sanity checks should become less and less necessary as end-to-end tests improve and integration tests are added.
+
+## Process
+
+* Test the master branch according to the tips mentioned above
+* Bump `lib/foreman_docker/version.rb` according to the Versioning policy specified in this document.
+* Run `changelog` and update the `Contributors` and `CHANGELOG` files with those generated by the
+script
+* `git add` your changes
+* `git commit -m "Bump version to x.y.z"`
+* `git push upstream master`
+* `git tag x.y.z`
+* `git push upstream x.y.z`
+* `git checkout x.0-stable`
+* `git pull --rebase upstream master`
```

Comment:
ah okay, I'd perhaps suggest removing --rebase or using `git merge --ff-only master` then so you don't accidentally rebase the stable branch if something _does_ change